src/main/java/uk/gov/hmcts/reform/migration/service/RespondentsDataMigrationService.java

```diff
@@ -0,0 +1,149 @@
+        data.put("respondents", null);
+
+        log.info("new case details: {}", caseDetails);
+    }
+
+    private List<Map<String, Object>> migrateRespondents(Map<String, Object> respondents) {
+        log.info("beginning to migrate respondents {}", respondents);
+
+        /// FIRST RESPONDENT
+
+        OldRespondent firstRespondent =
+            objectMapper.convertValue(respondents.get("firstRespondent"), OldRespondent.class);
+        Respondent migratedFirstRespondent = migrateIndividualRespondent(firstRespondent);
+
+        /// ADDITIONAL RESPONDENT
+
+        List<Map<String, Object>> additionalRespondents =
+            (List<Map<String, Object>>) objectMapper.convertValue(defaultIfNull(respondents.get("additional"), new ArrayList<>()), List.class);
+
+        List<Respondent> migratedRespondentCollection = additionalRespondents.stream()
+            .map(respondent ->
+                migrateIndividualRespondent(objectMapper.convertValue(respondent.get("value"), OldRespondent.class)))
+            .collect(toList());
+
+        // ADD FIRST RESPONDENT TO ADDITIONAL RESPONDENT LIST
+
+        migratedRespondentCollection.add(migratedFirstRespondent);
+
+        List<Map<String, Object>> newStructure = new ArrayList<>();
+
+        /// BUILD NEW STRUCTURE
+
+        migratedRespondentCollection.forEach(item -> {
+            Map map = objectMapper.convertValue(item, Map.class);
+
+            newStructure.add(ImmutableMap.of(
+                "id", UUID.randomUUID().toString(),
+                "value", map));
+        });
+
+        log.info("returning new structure {}", newStructure);
+
+        return newStructure;
+    }
+
+    private Respondent migrateIndividualRespondent(OldRespondent or) {
+        log.info("migrating respondent {}", or);
+
+        Address.AddressBuilder addressBuilder = Address.builder();
+        if (!isEmpty(or.getAddress())) {
+            addressBuilder.addressLine1(defaultIfBlank(or.getAddress().getAddressLine1(), null));
+            addressBuilder.addressLine2(defaultIfBlank(or.getAddress().getAddressLine2(), null));
+            addressBuilder.addressLine3(defaultIfBlank(or.getAddress().getAddressLine3(), null));
+            addressBuilder.postTown(defaultIfBlank(or.getAddress().getPostTown(), null));
+            addressBuilder.postcode(defaultIfBlank(or.getAddress().getPostcode(), null));
+            addressBuilder.county(defaultIfBlank(or.getAddress().getCounty(), null));
+            addressBuilder.country(defaultIfBlank(or.getAddress().getCountry(), null));
+        }
+        Address address = addressBuilder.build();
+
+        TelephoneNumber.TelephoneNumberBuilder telephoneNumberBuilder = TelephoneNumber.builder();
+        telephoneNumberBuilder.telephoneNumber(defaultIfBlank(or.getTelephone(), null));
+        TelephoneNumber telephoneNumber = telephoneNumberBuilder.build();
+
+        RespondentParty.RespondentPartyBuilder partyBuilder = RespondentParty.builder();
```

Comment:
If we decide to capture party type are we able to populate existing cases with current value (question to business) or we accept that it will be null for existing cases?